astropy/nddata/tests/test_utils.py

```diff
@@ -292,6 +292,14 @@ def test_extract_array_return_pos():
         assert new_pos == (expected, )
 
 
+def test_extract_array_nan_fillvalue():
+    if minversion(np, '1.20'):
```

Comment:
Sure you don't want to use `packaging` here like what Matt C did to the other tests for the sake of consistency?

Reply:
OK, I switched to use `packaging`.